src/ioctls/device.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The current structure of kvm-ioctls is to have  3 sub-modules in the `ioctls` module that correspond to the KVM ioctls: system, vm and vcpu. When helper structures are required we can do one of two things:
- if they're used by more than one sub-module (system, vm or vcpu) the definition can lie in ioctls/mod.rs
- it they're used by only one sub-module than they should be added to the sub-module that uses them.

I believe `DeviceFd` is used only by `VmFd` and if this is not going to change in the future I say we place this code there rather than creating a new file so we can keep the initial structure.

Reply:
Gotcha! Sounds good as it it. I checked the KVM API again, I missed the device ioctls entry before.